fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -801,6 +807,44 @@ public Long getValue() {
         } // end for backends
     }
 
+    public static void updateMemoryUsageMetrics() {
+        if (GAUGE_MEMORY_USAGE_STATS.size() ==
+                MemoryUsageTracker.MEMORY_USAGE.values().stream().mapToInt(Map::size).sum()) {
+            return;
+        }
+
+        List<GaugeMetricImpl<Long>> memoryUsageGauges = new ArrayList<>();
+        List<GaugeMetricImpl<Long>> objectCountGauges = new ArrayList<>();
+        MemoryUsageTracker.MEMORY_USAGE.forEach((moduleName, module) -> {
```

Comment:
Should refactor the code `initMemoryMetrics()` to use new memory tracking framework.

Reply:
Sure